Assets/MixedRealityToolkit-Examples.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 65c66d518b6f7d442b6cf5e5cced7329
```

Comment:
It might be best to rename the previously existing HoloToolkit, HoloToolkit-UnitTests, and HoloToolkit-Examples folder meta files, instead of having Unity generate new ones. I'm not sure if changing this guid under people might cause unintended issues, but it'd probably be best to (even ever so slightly) lower the amount of change we're introducing here as much as possible.

Reply:
Ok, I'll try resurrecting the old one's and renaming them.  
As these are root folders however, it's unlikely to cause issue.